src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/AssociatedAudio.kt

```diff
@@ -0,0 +1,18 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
```

Comment:
How about: ```package org.wycliffeassociates.otter.common.data.workbook```?

Reply:
I'm also up for renaming ```package org.wycliffeassociates.otter.common.data.model``` but idk what to. Perhaps core, but idk.

Reply:
I wondered if most of the existing `common.data.model` classes would go away after these new ones were integrated, and then everything could once again live in `common.data.model`.

Reply:
The workbook is great for the domain case of working within a particular translation project. This is mainly because concepts of Bible, Testament, etc are no longer relevant. 

It's not exactly great for the domain cases of creating a project, importing a resource container, etc.

I see them coexisting, and we may continue to create more domain specific data structures as time goes on, even though at first glance they may seem to closely resemble existing ones.

Reply:
OK, I renamed rxmodel -> workbook.  I also moved MimeType to the main model package.  I'll rename that one to core if you wish.